diff --git a/api_yamdb/api/views.py b/api_yamdb/api/views.py
index 45c156f..e73ea80 100644
--- a/api_yamdb/api/views.py
+++ b/api_yamdb/api/views.py
@@ -78,7 +78,6 @@ class TitleViewSet(viewsets.ModelViewSet):
     queryset = Title.objects.all().annotate(
         rating=Avg('reviews__score')
     ).order_by('name')
-    serializer_class = TitleSerializer
     permission_classes = (IsAdminOrReadOnly,)
     pagination_class = PageNumberPagination
     http_method_names = ['get', 'post', 'patch', 'delete']
diff --git a/api_yamdb/api_yamdb/settings.py b/api_yamdb/api_yamdb/settings.py
index 6522e40..ab7c337 100644
--- a/api_yamdb/api_yamdb/settings.py
+++ b/api_yamdb/api_yamdb/settings.py
@@ -10,7 +10,6 @@ BASE_DIR = Path(__file__).resolve().parent.parent
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = os.getenv('SECRET_KEY')
-SECRET_KEY = 'p&l%385148kslhtyn^##a1)ilz@4zqj=rq&agdol^##zgl9(vs'
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
